lib/v2/inewsweek/index.js

```diff
@@ -0,0 +1,49 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const iconv = require('iconv-lite');
+
+const rootUrl = 'http://news.inewsweek.cn';
+
+module.exports = async (ctx) => {
+    const channel = ctx.params.channel;
+    const url = `${rootUrl}/${channel}`;
+    const response = await got(url, {
+        responseType: 'buffer',
+    });
+    const $ = cheerio.load(iconv.decode(response.data, 'gbk'));
+    const items = await Promise.all(
+        $('div.grid-item')
+            .toArray()
+            .map((item) => {
+                item = $(item);
+                const href = item.find('a').attr('href');
+                const articleLink = `${rootUrl}${href}`;
+                return ctx.cache.tryGet(articleLink, async () => {
+                    const response = await got(articleLink, {
+                        responseType: 'buffer',
+                    });
+
+                    const $ = cheerio.load(iconv.decode(response.data, 'gbk'));
+                    const fullText = $('div.contenttxt').html();
+                    const time = parseDate(
+                        $('div.editor')
+                            .html()
+                            .split(/(\s\s+)/)[2]
+                    );
```

Comment:
```suggestion
                    const time = timezone(parseDate(
                        $('div.editor')
                            .html()
                            .split(/(\s\s+)/)[2]
                    ), 8);
```
You have to make sure the timezone is correct as well. `parseDate` will treat the date-time string parsed as UTC time.
```js
const timezone = require('@/utils/timezone');
```

Reply:
Since the article is published in Mainland China, the publish time is implicitly treated as UTC+8 (Beijing Time)

Reply:
Since articles are published in Mainland China, the publish time is implicitly treated as UTC+8 (Beijing Time)